<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>718</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainMenu">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1371</width>
        <height>711</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QPushButton" name="riskManagementButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Risk Management</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="analysisButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Analysis</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="strategiesButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Strategies</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="exitButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>EXIT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="portfolioButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Portofoliu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="apiMenu">
     <widget class="QWidget" name="gridLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>180</y>
        <width>1361</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <property name="spacing">
        <number>1</number>
       </property>
       <item row="3" column="0">
        <widget class="QLineEdit" name="secretApiKeyInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 255, 49);</string>
         </property>
         <property name="text">
          <string>API Secret Key</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="apiKeyInput">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>50</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(238, 255, 49);</string>
         </property>
         <property name="text">
          <string>API Key</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="apiLoginButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 242, 64);</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="portfolioMenu">
     <property name="maximumSize">
      <size>
       <width>1366</width>
       <height>718</height>
      </size>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1371</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="2" column="1">
        <widget class="QPushButton" name="SellOrder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Sell order</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="AddTransactionOrder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Add transaction order</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="BuyOrder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Buy order</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="Balance">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Balance </string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="CancelTransactionOrder">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Cancel transaction order</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="portfolioBackButton">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>660</y>
        <width>130</width>
        <height>45</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="analysisMenu">
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1371</width>
        <height>81</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item row="2" column="1">
        <widget class="QPushButton" name="ViewTransactionHistory">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>View transaction history</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="GenerateReport">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Generate report </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="analysisBackButton">
      <property name="geometry">
       <rect>
        <x>1110</x>
        <y>620</y>
        <width>130</width>
        <height>45</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>45</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 62, 58);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="strategiesMenu">
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1371</width>
        <height>791</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="2" column="0">
        <widget class="QPushButton" name="strategiesBackButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(43, 43, 43);</string>
         </property>
         <property name="text">
          <string>Go back</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="predefinedStrategiesButton_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(43, 43, 43);</string>
         </property>
         <property name="text">
          <string>Predefined Strategies </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="userStrategiesButton_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(43, 43, 43);</string>
         </property>
         <property name="text">
          <string>Your strategies</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="riskManagementMenu">
     <widget class="QWidget" name="gridLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1361</width>
        <height>771</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="5">
        <widget class="QToolButton" name="portfolioButton_10">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Buy Order</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="portfolioButton_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Sell market</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="portfolioButton_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Stop loss </string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QToolButton" name="portfolioButton_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Buy Limit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="portfolioButton_9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Take Profit</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="portfolioButton_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Buy market</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QToolButton" name="riskManagementBackButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>130</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(42, 62, 58);</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
